src/main/java/com/johnwesthoff/bending/logic/World.java

```diff
@@ -28,60 +13,51 @@
 import com.johnwesthoff.bending.entity.WaterEntity;
 import com.johnwesthoff.bending.util.network.ResourceLoader;
 
+import java.awt.*;
+import java.awt.geom.AffineTransform;
+import java.awt.image.BufferedImage;
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Random;
+import java.util.concurrent.CopyOnWriteArrayList;
+
+import static com.johnwesthoff.bending.Constants.*;
+
 /**
- *
  * @author John
  */
 public class World implements Serializable {
-    public int incX, incY;
+    public int incX, incY, floatiness = 0, viewX = 0, viewY = 0, viewdX = 0, viewdY = 0, flowCount = 0, maxFlow = 5000;
     public Random random = new Random();
     public CollisionChecker ground;
     public final CopyOnWriteArrayList<Entity> entityList = new CopyOnWriteArrayList<>();
     public float x = 450, y = 0;
-    public int floatiness = 0;
     public short status = 0;
     /*
      * ~~~~~~~~~~~~~~~~~~~~~Status values~~~~~~~~~~~~~~~~~~~~~ Index: Effect: 0
      * Flaming 1 Shocked 8 Invisible 9 Dark Cloud
-     * 
-     * 
+     *
      */
-    public static final int ST_FLAMING = 0b1;
-    public static final int ST_SHOCKED = 0b10;
-    public static final int ST_INVISIBLE = 0b100000000;
-    public static final int ST_DRAIN = 0b1000000000;
-    public static final int AURA_RADIUS = 96;
-    public int wIdTh = 900, hEigHt = 900;
-    public int viewX = 0, viewY = 0, viewxX = 0, viewyY = 0, viewdX = 0, viewdY = 0;
+
     public final ArrayList<Player> playerList = new ArrayList<>();
     public boolean serverWorld = false, dead = false;
     // public Image terrain;
     public BufferedImage Grass, Sky, Sand, Stone, Ice, Night, Crystal, Ether, Bark;// at the moon
     // public Graphics2D G2D;
-    public double vspeed = 0;
-    public int mouseX = 150, mouseY = 0, V, previousX = 0, previousY = 0, pressX, pressY;
-    public double move = 0;
-    public boolean keys[] = new boolean[3200];
+    public double vspeed = 0, move = 0, fr;
+    public int mouseX = 150, mouseY = 0, previousX = 0, previousY = 0, pressX, pressY;
+    public boolean[] keys = new boolean[3200];
     public float jump = 0;
     public boolean MB1, MB2, MB3;
-    public int landTexSize = 256;
-    public int FPS = 0;
-    public static int head = 26;
-    public static int body = 13;
-    public int flowCount = 0, maxFlow = 5000;
+
     public TexturePaint skyPaint, grassPaint, sandPaint, stonePaint, barkPaint, icePaint, nightPaint;
-    public static final byte AIR = 0, GROUND = 1, WATER = 2, OIL = 3, LAVA = 4, SAND = 5, STONE = 6, TREE = 7, ICE = 8,
```

Comment:
It looks like these were moved to Constants, yet within this file they don't seem to reference those.

Reply:
Yes they are being referenced from the Constants.java file. Do take a look at the imports.

Reply:
Ah I see now, thanks.